internal/checkers/checkers_registry.go

```diff
@@ -12,6 +12,7 @@ var registry = checkersRegistry{
 	{factory: asCheckerFactory(NewEmpty), enabledByDefault: true},
 	{factory: asCheckerFactory(NewLen), enabledByDefault: true},
 	{factory: asCheckerFactory(NewCompares), enabledByDefault: true},
+	{factory: asCheckerFactory(NewNegativePositive), enabledByDefault: true},
```

Comment:
looks like it should be before Compares

Reply:
Indeed

Reply:
I'm no longer sure now

Please look at https://github.com/Antonboom/testifylint/pull/78#discussion_r1561432681